src/dftbp/math/lapackroutines.F90

```diff
@@ -320,6 +330,89 @@ subroutine gesv_dcomplex(aa, bb, nEquation, nSolution, iError)
   end subroutine gesv_dcomplex
 
 
+#:for KIND, NAME, PRC, LABEL, PRF in [('real', 'symmetric', 'rsp', 'real', 'sposv'),&
```

Comment:
```suggestion
#:for TYPE, NAME, KIND, LABEL, PRF in [('real', 'symmetric', 'rsp', 'real', 'sposv'),&
```
In the ususal Fortran nomenclature would be `TYPE(KIND)`, so we should probably name the preprocessor variables also accordingly

Reply:
The *tri and *trf routines in the same file do not follow that convention either, so should be I change those and document here or in another PR after this?

Reply:
> The *tri and *trf routines in the same file do not follow that convention either, so should be I change those and document here or in another PR after this?

Looking at your comment on #1670 I think a new PR to fix those two...